homeassistant/components/light/__init__.py

```diff
@@ -41,6 +42,18 @@
 DATA_PROFILES: HassKey[Profiles] = HassKey(f"{DOMAIN}_profiles")
 
 
+class LightState(StrEnum):
+    """Light entity states."""
+
+    ON = "on"
+    OFF = "off"
+
+
+# The STATE_ON constant is deprecated as of Home Assistant 2024.10
+# Please use the LightState enum instead.
+_DEPRECATED_STATE_ON = DeprecatedConstantEnum(LightState.ON, "2025.10")
```

Comment:
Only `STATE_ON` was imported, so there is no need to deprecate `STATE_OFF` here.